lib/src/main/java/grpcbridge/Bridge.java

```diff
@@ -116,33 +132,40 @@ public HttpResponse handle(HttpRequest httpRequest) {
             if (optionalCall.isPresent()) {
                 RpcCall call = optionalCall.get();
                 ListenableFuture<RpcMessage> response = call.execute();
-                JsonFormat.Printer printer = route.getPrinter();
-                return Futures.transform(response, new RpcToHttpMessage(printer));
+                String preferredResponseType = route.preferredResponseType();
```

Comment:
I don't think we should do it per route. Should be controlled by the client instead, with the "Accept" header. If there is no accept header we should just have default to JSON.

Reply:
This is driven via the client's `accept` header as you've proposed, the problem is that sometimes a client might not specify the accept header correctly, often clients will specify accept as `*/*` but will only really accept `text/xml` or something else. This is just a way to set a preferred / default for that service. It could be done in another way i guess. I'm open to suggestions :-)

Reply:
I guess another way would be to only add the specific parser you want to the bridge builder, then we don't need to do this check